Doc/library/time.rst

```diff
@@ -708,6 +626,101 @@ The module defines the following functions and data items:
       ('EET', 'EEST')
 
 
+Clock ID Constants
+------------------
+
+.. data:: CLOCK_HIGHRES
+
+   The Solaris OS has a CLOCK_HIGHRES timer that attempts to use an optimal
```

Comment:
@berkerpeksag  Thank you for clarifying.  I had tried to do that, but, when I did, the tick marks show up in the HTML (after running make html).  It doesn't show up here when I cut and paste from the HTML because of Markdown, so I don't know how to show you what it looks like.